service/resources/bluebeam.py

```diff
@@ -182,6 +182,17 @@ def create_project(access, project_name):
     print("Created project id:{0}".format(idee))
     return idee
 
+def sanitize_project_name(project_name):
+    """
+        returns new project name after illegal chars have been removed
+        https://support.bluebeam.com/articles/unsupported-characters-in-studio/
+    """
+    #allow alpha numeric, underscore, period, spaces
+    new_name = re.sub(r"[^\w. #]", "", project_name)
```

Comment:
What do you think of this suggestion? does \w include dashes? 
```suggestion
    new_name = re.sub(r"[^\w. #]", " ", project_name)
```

Reply:
You want to replace illegal chars with spaces?  In the case of "TEST 1 SOUTH VAN NESS *", that will result with a project name with trailing space.
\w includes _ but not -

Reply:
then for trailing and possibily leading space, maybe we can do a trim?

Reply:
https://github.com/SFDigitalServices/bluebeam-microservice/pull/53/commits/aefe02a81990ff107022cd0184f4676bd4d4d440